src/python/qepsi4/_psi4.py

```diff
@@ -185,13 +205,13 @@ def get_ham_from_psi4(
 
     if n_active_extract is None:
         trf_mat = wfn.Ca()
-        n_active_extract = wfn.nmo() - n_core_extract
+        n_active_extract = wfn.nmo() - n_core_extract # - wfn.nfzvpi.sum() ?
     else:
         # If orbs is given, it allows us to perform the two-electron integrals
         # transformation only in the space of active orbitals. Otherwise, we
         # transform all orbitals and filter them out in the get_ham_from_integrals
         # function
-        if orbs is None:
+        if orbs is None: # If n_acitve_extract is not None - how could possibly be orbs == None?
```

Comment:
Based on what I see in `run_psi4`  the orbs argument is always passed to `get_ham_from_psi4` if  `n_acitve_extract` is not None. Should I just remove this check or do you think it would be better to keep it?

Reply:
I think we can keep it here for now, in case somebody uses this function somewhere else (which is a bit unlikely, I admit). We should probably refactor get_hamiltonian at some point to stop using `orbs` at all. I made a tracker story for that: https://www.pivotaltracker.com/story/show/175677313
It's not urgent at all but it's probably also not much effort.

Reply:
Gotcha. Thanks for making the story! I was actually suggesting removing `if orbs is None:` since it would always evaluate to `False` in the above context but this does not apply to the latest version of the code.